script/test_Dockerfile

```diff
@@ -16,16 +16,24 @@ RUN apt-get update && apt-get install -y \
     python-minimal \
     --no-install-recommends
 
+# install bats 
+RUN cd /tmp \
```

Comment:
I think he meant "we should add it to the Jenkins jobs in another PR". I don't like pulling in the latest version of bats from HEAD when doing testing -- that doesn't strike me as being a particularly good idea.